qiita_db/test/test_meta_util.py

```diff
@@ -34,17 +34,13 @@ def tearDown(self):
 
     def _set_artifact_private(self):
         id_status = qdb.util.convert_to_id('private', 'visibility')
-        with qdb.sql_connection.TRN:
-            qdb.sql_connection.TRN.add(
-                "UPDATE qiita.artifact SET visibility_id = %d" % id_status)
-            qdb.sql_connection.TRN.execute()
+        qdb.sql_connection.encapsulated_query(
+            "UPDATE qiita.artifact SET visibility_id = %d" % id_status)
```

Comment:
shouldn't id_status be an argument to encapsulated_query?

... there's a few other instances here. If I'm missing something then feel free to disregard these messages.

Reply:
Yeah, we normally use the parameters as arguments when adding a parameter to have safe data type conversions in the query; however, if a developer is sure about the data type you can do this. Note that this is done based on the developer and didn't change any of the code to keep it simple (and cause it was out of the scope of this issue/PR).

Reply:
Alright, that sounds fair enough.